code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -131,6 +159,17 @@
 
 /mob/living/simple_animal/hostile/giant_spider/Initialize(mapload, atom/parent)
 	get_light_and_color(parent)
+	add_language(LANGUAGE_GREIMORIAN)
+	add_language(LANGUAGE_GREIMORIAN_HIVEMIND)
+	. = ..()
+
+/mob/living/simple_animal/hostile/giant_spider/nurse/servant/Initialize()
+	add_verb(src, /mob/living/proc/ventcrawl)
+	var/number = rand(1000,9999)
+	name = initial(name) + " ([number])"
+	real_name = name
+	if(playable && !ckey && !client)
+		SSghostroles.add_spawn_atom("servant", src)
```

Comment:
Need to remove it on `/Destroy` implementation

Reply:
Handled